spec/voting_spec.rb

```diff
@@ -54,4 +57,25 @@
     end
   end
 
+  context "when a topic is moved to a category" do
+    let(:admin) { Fabricate(:admin) }
+    let(:post0) { Fabricate(:post, topic: topic0, post_number: 1) }
+    let(:post1) { Fabricate(:post, topic: topic1, post_number: 1) }
+    
+    before do
+      category1.custom_fields["enable_topic_voting"] = "true"
+      category1.save!
+      Category.reset_voting_cache
+    end
+
+    it "enqueus a job to reclaim votes if voting is enabled for the new category" do
+      PostRevisor.new(post1).revise!(admin, category_id: category1.id)
+      expect(Jobs::VoteReclaim.jobs.first["args"].first["topic_id"]).to eq(post1.reload.topic_id)
```

Comment:
A problem with this style of test is that it does not properly test if the thing works, the job may fire and crash for all you know... the better way is just to test that votes actually get reclaimed.

Can you change that?